support/index.adoc

```diff
@@ -125,11 +125,11 @@ endif::openshift-rosa,openshift-dedicated[]
 ** Investigate why user-defined metrics are unavailable.
 ** Determine why Prometheus is consuming a lot of disk space.
 
-* xref:../logging/cluster-logging.adoc#cluster-logging[Logging issues]: A cluster administrator can follow the procedures on the troubleshooting page for OpenShift Logging issues. Check the following to resolve logging issues:
+* xref:../logging/cluster-logging.adoc#cluster-logging[Logging issues]: A cluster administrator can follow the procedures in the "Support" and "Troubleshooting logging" sections to resolve logging issues:
 
-** xref:../logging/troubleshooting/cluster-logging-cluster-status.adoc#cluster-logging-cluster-status[Status of the Logging Operator].
-** xref:../logging/troubleshooting/cluster-logging-cluster-status.adoc#cluster-logging-cluster-status[Status of the Log store].
-** xref:../logging/logging_alerts/troubleshooting-logging-alerts.adoc#troubleshooting-logging-alerts[Troubleshooting logging alerts].
-** xref:../logging/cluster-logging-support.adoc#cluster-logging-support-must-gather_cluster-logging-support[Information about your OpenShift logging environment using `oc adm must-gather` command].
+** xref:../logging/troubleshooting/cluster-logging-cluster-status.adoc#cluster-logging-clo-status_cluster-logging-cluster-status[Viewing the status of the {clo}].
+** xref:../logging/troubleshooting/cluster-logging-cluster-status.adoc#cluster-logging-clo-status-comp_cluster-logging-cluster-status[Viewing the status of {logging} components].
+** xref:../logging/troubleshooting/troubleshooting-logging-alerts.adoc#troubleshooting-logging-alerts[Troubleshooting logging alerts].
+** xref:../logging/cluster-logging-support.adoc#cluster-logging-support-must-gather_cluster-logging-support[Information about your logging environment using `oc adm must-gather` command].
```

Comment:
Wonder if this link could be/should directed to this section of the doc given the subject matter? https://70681--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/cluster-logging-support#cluster-logging-must-gather-collecting_cluster-logging-support?